libraries/AP_Math/control.cpp

```diff
@@ -137,11 +94,11 @@ void update_pos_vel_accel_xy(Vector3f& pos, Vector3f& vel, const Vector3f& accel
  The time constant defines the acceleration error decay in the kinematic path as the system approaches constant acceleration.
  The time constant also defines the time taken to achieve the maximum acceleration.
  The time constant must be positive.
- The function alters the input velocity to be the velocity that the system could reach zero acceleration in the minimum time.
+ The function alters the input accel to be jerk limited
```

Comment:
The function alters the variable accel to follow a jerk limited kinematic path to the variable accel_input

Reply:
fixed